tests/bin/non-github-doc-uri/run.t

```diff
@@ -0,0 +1,162 @@
+    => exists whatever.opam
+    [ OK ] File opam is present.
+    -: exec: opam lint -s whatever.opam
+    [ OK ] lint opam file whatever.opam.
+    [ OK ] opam field description is present
+    [ OK ] opam fields homepage and dev-repo can be parsed by dune-release
+    [FAIL] opam field doc cannot be parsed by dune-release
+    [ OK ] lint _build/whatever-0.1.0 success
+    
+    [-] Building package in _build/whatever-0.1.0
+    => chdir _build/whatever-0.1.0
+    -: exec: dune build -p whatever
+    [ OK ] package builds
+    
+    [-] Running package tests in _build/whatever-0.1.0
+    => chdir _build/whatever-0.1.0
+    -: exec: dune runtest -p whatever
+    [ OK ] package tests
+    -: rmdir _build/whatever-0.1.0
+    
+    [+] Distribution for whatever 0.1.0
+    [+] Commit ...
+    [+] Archive _build/whatever-0.1.0.tbz
+
+We do the whole dune-release process
+
+(1) distrib
+
+    $  yes | dune-release distrib --dry-run
+    [-] Building source archive
+    => rmdir _build/whatever-0.1.0.build
+    -: exec: git --git-dir .git rev-parse --verify 0.1.0
+    => exec: git --git-dir .git show -s --format=%ct 0.1.0^{commit}
+    => exec: git --git-dir .git clone --local .git _build/whatever-0.1.0.build
+    => exec:
+         git --git-dir _build/whatever-0.1.0.build/.git --work-tree   _build/whatever-0.1.0.build/ checkout --quiet -b dune-release-dist-0.1.0   0.1.0
+    => chdir _build/whatever-0.1.0.build
+       [in _build/whatever-0.1.0.build]
+    -: exec: dune subst
+    -: write whatever.opam
+    => exec: bzip2
+    -: rmdir _build/whatever-0.1.0.build
+    [+] Wrote archive _build/whatever-0.1.0.tbz
+    => chdir _build/
+       [in _build]
+    => rmdir _build/whatever-0.1.0
+    => exec: tar -xjf whatever-0.1.0.tbz
+    
+    [-] Linting distrib in _build/whatever-0.1.0
+    => chdir _build/whatever-0.1.0
+       [in _build/whatever-0.1.0]
+    => exists ./README
+    [ OK ] File README is present.
+    => exists ./LICENSE
+    [ OK ] File LICENSE is present.
+    => exists ./CHANGES.md
+    [ OK ] File CHANGES is present.
+    => exists whatever.opam
+    [ OK ] File opam is present.
+    -: exec: opam lint -s whatever.opam
+    [ OK ] lint opam file whatever.opam.
+    [ OK ] opam field description is present
+    [ OK ] opam fields homepage and dev-repo can be parsed by dune-release
+    [FAIL] opam field doc cannot be parsed by dune-release
+    [ OK ] lint _build/whatever-0.1.0 success
+    
+    [-] Building package in _build/whatever-0.1.0
+    => chdir _build/whatever-0.1.0
+    -: exec: dune build -p whatever
+    [ OK ] package builds
+    
+    [-] Running package tests in _build/whatever-0.1.0
+    => chdir _build/whatever-0.1.0
+    -: exec: dune runtest -p whatever
+    [ OK ] package tests
+    -: rmdir _build/whatever-0.1.0
+    
+    [+] Distribution for whatever 0.1.0
+    [+] Commit ...
+    [+] Archive _build/whatever-0.1.0.tbz
+
+(2) publish
+
+    $  yes | dune-release publish --dry-run
```

Comment:
How about making this `dune-release publish doc --dry-run` so that the test don't break if we ever decide to change the order of publishing?

Reply:
done